src/sanitizer.js

```diff
@@ -203,9 +204,9 @@ export function sanitizeHtml(html, diffing) {
       emit('<');
       emit(tagName);
       for (let i = 0; i < attribs.length; i += 2) {
-        const attrName = attribs[i];
+        const attrName = attribs[i].toLowerCase();
```

Comment:
This can be reverted right?

Reply:
yes. Done

Reply:
Right, I thought I was correcting logic here as the documentation mentioned lowerCase

     * @param {string} attrName Lowercase attribute name.

and I didn't see this value being lowercased but all attr names are case insensitive so this is not needed. Fixed.